openmdao/drivers/doe_driver.py

```diff
@@ -73,6 +77,8 @@ def __init__(self, generator=None, **kwargs):
         self._name = ''
         self._problem_comm = None
         self._color = None
+        self._num_model_evals = 0
+        self._num_deriv_evals = 0
```

Comment:
I don't see where these attributes are consumed, they seem redundant with the decorator.. am I missing something?

Reply:
Bah you're right.